Resources/Audio/Jukebox/attributions.yml

```diff
@@ -36,3 +36,14 @@
   copyright: "Space Asshole by Chris Remo is used with special permission from the author, under the condition that the project remains non-commercial and open source. The author also requested that a link to his bandcamp be included: https://chrisremo.bandcamp.com/"
   source: "https://idlethumbs.bandcamp.com/track/space-asshole"
   # The source is a direct link the the track, but not the "main" bandcamp of the author. Hence the link is also included separately in the copyright.
+  
+- files: ["Nuclear.ogg"]
+  license: "i dunno"
+  copyright: "2014 Mercury Records Limited"
+  source: "https://www.youtube.com/watch?v=R_TbF5m5KpU"
+  
+ - files: ["Cuban Pete.ogg"]
+  license: "i dunno"
+  copyright: "the mask? man i got no clue"
+  source: "https://www.youtube.com/watch?v=jA4u5Pur5KA"
```

Comment:
okay.

1. this isn't going to compile correctly because the licensing is messed up.
2. I'm 90% certain both of these songs are copyrighted without the ability to give credit, meaning even with credit it still breaks copyright law.